vueapp/src/locales/en.json

```diff
@@ -15,7 +15,8 @@
     "titleSetPasscode": "Set Passcode",
     "buttonReset": "Reset All Settings & Tiles",
     "buttonExport": "Export Settings & Tiles",
-    "buttonImport": "Import Settings & Tiles"
+    "buttonImport": "Import Settings & Tiles",
+    "displayTapCount": "Mostrar uso de los mosaicos"
```

Comment:
If you could add the English translation here that would be awesome. or i can grab it in another pr later.

Reply:
I totally missed it, sorry. Already fixed

Reply:
That was quick! Looks great. Thank you again!